plugins/woocommerce-blocks/assets/js/atomic/blocks/product-elements/product-details/style.scss

```diff
@@ -1,38 +1,72 @@
-.wp-block-woocommerce-product-details {
-	ul.wc-tabs {
-		list-style: none;
-		padding: 0 0 0 1em;
-		margin: 0 0 1.618em;
-		overflow: hidden;
-		position: relative;
-		border-bottom: 1px solid $gray-200;
+/*
+ * Minimal style.
+ */
+body .wp-block-woocommerce-product-details.is-style-minimal {
+	ul.tabs.wc-tabs {
+		padding: 0;
+		border-bottom-width: 0;
+		border-bottom-color: inherit; /* Backward compatibility for TT3, TT2. */
+
+		&::after,
+		&::before {
+			display: none;
+		}
 
 		li {
-			border: 1px solid $gray-200;
-			display: inline-block;
-			position: relative;
-			z-index: 0;
-			border-radius: $universal-border-radius $universal-border-radius 0 0;
+			color: inherit;
+			background: transparent;
+			border-color: transparent;
+			border-style: solid;
+			border-width: 0;
+			border-bottom-width: 2px;
+			border-radius: 0;
 			margin: 0;
-			padding: 0.5em 1em;
+			padding: 0 1em;
+			display: inline-block;
+			float: none; /* Backward compatibility for TT3, TT2. */
+			font-weight: bold;
+			opacity: 0.65;
 
 			a {
-				display: inline-block;
-				font-weight: 700;
+				padding: 0.5em 0;
+				color: inherit;
+				border: none;
+				text-shadow: none;
 				text-decoration: none;
+				display: inline-block;
+			}
 
-				&:hover {
-					text-decoration: none;
-				}
+			&:hover {
+				opacity: 1;
```

Comment:
Oh, one more thing. We might need to add `color: inherit` here too, otherwise the default WooCommerce styles leak in here and the hovered tab is displayed gray:

[](https://github.com/woocommerce/woocommerce/assets/3616980/33189952-14b2-40c8-9067-b482dc24732f)

Reply:
Thanks for catching this, @Aljullu! I only tested the latest changes using TT3, which has much less specific styles!

Reply:
Done in https://github.com/woocommerce/woocommerce/pull/42965/commits/fd915113de1dce29e0e26559467806df1c9cef8b